app/code/Magento/DataExporter/Model/Indexer/LegacyFeedIndexer.php

```diff
@@ -0,0 +1,361 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\DataExporter\Model\Indexer;
+
+use Magento\DataExporter\Export\Processor;
+use Magento\DataExporter\Model\FeedPool;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Select;
+use Magento\Framework\Indexer\ActionInterface as IndexerActionInterface;
+use Magento\Framework\Mview\ActionInterface as MviewActionInterface;
+
+/**
+ * Product export feed indexer class
+ */
+class LegacyFeedIndexer implements IndexerActionInterface, MviewActionInterface
```

Comment:
What makes a feed "legacy"? Is it because the queries are in code vs in XML? 

If so, could it be possible to use either XML query or an in code query? There may be cases where XML is more limiting and a more complex query is needed.

Reply:
I have trouble finding a proper name, this particular class coupled to products and can not be reused, but in the future, I do want to have it at all. for now, removing could cause unnecessary refactoring ...